src/core/cmd.rs

```diff
@@ -4,6 +4,60 @@
 use xrl::ViewId;
 
 use std::str::FromStr;
+use serde::{Deserialize, Serialize};
+
+#[allow(non_camel_case_types)]
```

Comment:
I agree. To be honest, at the time, I didn't know about the feature to give serde a different name from the structs-name. Found out later and didn't change it because I wanted to spent the little time I had on features instead.

Reply:
done